irr/src/OpenGL/Renderer2D.cpp

```diff
@@ -62,6 +62,18 @@ void COpenGL3Renderer2D::OnSetMaterial(const video::SMaterial &material,
 	f32 Thickness = (material.Thickness > 0.f) ? material.Thickness : 1.f;
 	setPixelShaderConstant(ThicknessID, &Thickness, 1);
 
+	{
+		ProjectionID = getPixelShaderConstantID("uProjection");
```

Comment:
you can move this to the ctor

Reply:
No because the ctor does not appear to be called on screen resize.